portal/.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Remove the cache files. Add it git ignore.

Reply:
there is already a gitignore file in portal like osmox, added a gitignore file in root directory just like the one in osmox